CHANGELOG.md

```diff
@@ -2,6 +2,25 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+## Mapbox Navigation SDK 2.1.0-alpha.1 - September 23, 2021
+
+For details on how v2 differs from v1 and guidance on migrating from v1 of the Mapbox Navigation SDK for Android to the v2 public preview, see [2.0 Navigation SDK Migration Guide](https://github.com/mapbox/mapbox-navigation-android/wiki/2.0-Navigation-SDK-Migration-Guide).
+
+### Changelog
+
+#### Bug fixes and improvements
+- Moved `zLevel` from `RouteProgress` to `MapMatcherResult` [#4849](https://github.com/mapbox/mapbox-navigation-android/pull/4849)
```

Comment:
```suggestion
- Exposed MapboxNavigation#getZLevel that can be used with RouteOptions#layers to influence the layer of the road from where the route starts. [#4849](https://github.com/mapbox/mapbox-navigation-android/pull/4849)
```

Reply:
1 hour change, coming up lol

Reply:
just kidding, thanks for the comment here! Also needed to resolve conflicts from the v2.0.0-rc.3 update

This one should be ready

Reply:
https://github.com/mapbox/mapbox-navigation-android/pull/4904 :pray: